extensions/amp-form/0.1/form-validators.js

```diff
@@ -74,6 +75,12 @@ export class FormValidator {
 
     /** @protected @const {!Document} */
     this.doc = /** @type {!Document} */ (form.ownerDocument);
+
+    /**
+     * Tribool indicating last known validity of form.
+     * @private {boolean|null}
+     */
+    this.formValidity_ = null;
```

Comment:
Is there an extra benefit to making this a tribool vs setting it to the initial validity of the form as returned by `form.checkValidity()`, or an enum?

[Relevant : P](https://thedailywtf.com/articles/What_Is_Truth_0x3f_)